### DISCUSSION TITLE
Particle out of bounds fix

### DISCUSSION THREAD

**Discussion by sikora507** [hunar4321/particle-life#29] [Ideas]
I saw your comment in ofApp.cpp about bouncing off the edges which needs to be fixed.
![image](https://user-images.githubusercontent.com/4272052/188281089-725886c4-1614-445b-ae1f-c8021f05ae78.png)

I've written the app in GO and this fixes it:
![image](https://user-images.githubusercontent.com/4272052/188281056-efd0581e-50ba-42f4-a73d-1b043a965a67.png)

Just don't check for < 0 and > max in single if.
If < 0, set position to 0, reverse velocity
If > max, set position to max, reverse velocity

**Comment by hunar4321** [comment]
Thanks a lot
